src/lib/modules/server/sessions/qwen-reader.ts

```
@@ -305,6 +308,29 @@ function readQwenTextBounded(filePath: string): string {
+    }
+    try {
+      const msg = qwenLineToMessage(JSON.parse(trimmed) as Record<string, unknown>);
+      if (msg) {
+        messages.push(msg);
+      }
+    } catch {
+      // skip malformed line
+    }
+  }
+  return messages;
+}
```

ESLint

[error] Expected "readQwenMessages" to come before "readQwenTextBounded".

(perfectionist/sort-modules)